compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -157,7 +157,8 @@ object RefChecks {
    *    1.8.3  M is of type ()S, O is of type []T and S <: T, or
    *    1.9.1  If M is erased, O is erased. If O is erased, M is erased or inline.
    *    1.9.2  If M or O are extension methods, they must both be extension methods.
-   *    1.10   If O is inline, M must be inline
+   *    1.10.1. If O is inline, M must be inline
+   *    1.10.2. If M is inline, M must not be deferred
```

Comment:
Deferred inlines should be alllowed!

Reply:
This is not an abstract inline. It is for an inline abstract override .

```scala
class A:
  def a: Int = 3

class B extends A:
  override inline def a: Int
```

```scala
class A:
  inline def a: Int = 3

class B extends A:
  override inline def a: Int
```

Reply:
I removed it